Solidner/Views/Onboarding/BabyBirthDateView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  BabyBirthDateView.swift
+//  Solidner
+//
+//  Created by 황지우2 on 2023/11/13.
+//
+
+import SwiftUI
+
+struct BabyBirthDateView: View {
+    let bigTitleColor = Color(#colorLiteral(red: 0.0834152922, green: 0.0834152922, blue: 0.0834152922, alpha: 1))
+    let smallTitleColor = Color(#colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)).opacity(0.6)
+    let cakeLabelFontSize = 32.0
+    let onboardingTitlesTopPadding = 10.0
+    let datePickerTopPadding = 70.0
+    @State var babyName = "찍무"
+    @State var babyBirthDate = Date()
```

Comment:
```suggestion
    @State private var babyName = "찍무"
    @State private var babyBirthDate = Date()
```

p3;
`@State` 는 private 으로 선언해주세요!